packages/eslint-plugin-pf-codemods/src/rules/v6/cardRemoveVariousProps/card-remove-various-props.test.ts

```diff
@@ -0,0 +1,77 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./card-remove-various-props";
+
+interface InvalidTest {
+  code: string;
+  output: string;
+  errors: { message: string }[];
+}
+
+const removedBooleanProps = [
+  "isSelectableRaised",
+  "isDisabledRaised",
+  "hasSelectableInput",
+  "isFlat",
+  "isRounded",
+];
+const removedStringProps = ["selectableInputAriaLabel"];
+const removedFunctionProps = ["onSelectableInputChange"];
+
+const removedBooleanPropTests = removedBooleanProps.map((prop) => ({
+  code: `import { Card } from '@patternfly/react-core'; <Card ${prop} />`,
+  output: `import { Card } from '@patternfly/react-core'; <Card  />`,
+  errors: [{ message: `The ${prop} prop has been removed` }],
+}));
+
+const removedStringPropTests = removedStringProps.reduce((acc, prop) => {
+  return [
+    ...acc,
+    {
+      code: `import { Card } from '@patternfly/react-core'; <Card ${prop}='foo bar' />`,
+      output: `import { Card } from '@patternfly/react-core'; <Card  />`,
+      errors: [{ message: `The ${prop} prop has been removed` }],
+    },
+    {
+      code: `import { Card } from '@patternfly/react-core'; const foo = "foo bar"; <Card ${prop}={foo} />`,
+      output: `import { Card } from '@patternfly/react-core'; const foo = "foo bar"; <Card  />`,
+      errors: [{ message: `The ${prop} prop has been removed` }],
+    },
+  ];
```

Comment:
Not opposed to including tests like this for rename/removal of props, just curious if you foresee not testing for it being an issue? Since the helper itself doesn't really care what's passed into the prop, just its existence.

Reply:
Sooooo long term I actually think we should change this by having dedicated tests for the helpers themselves. Until we can do that though I feel like these additional tests are worth it on the actual rule side.

Reply:
Yeah I was thinking that as well  re: tests for helpers. I'm good with keeping these tests, though, as it doesn't really hurt.